array_family/flex_ext.py

```diff
@@ -469,6 +471,48 @@ def subsort(self, key0, key1, reverse=False):
             ref_tmp.sort(key1, reverse)
             self[min(val) : (max(val) + 1)] = ref_tmp
 
+    def match_by_hkle(
+        self, other: dials_array_family_flex_ext.reflection_table
+    ) -> Tuple[cctbx.array_family.flex.size_t, cctbx.array_family.flex.size_t,]:
+        """
+        Match reflections with another set of reflections by the h, k, l
+        and entering values. Uses pandas dataframe merge method to match
+        the columns: assumes the key h, k, l, e is unique which is false
+        if > 360 degree rotation.
+
+        :param other: reflection table to match against
+        :return: indices in self, indices in other for matches
```

Comment:
I think elsewhere we've tended to use something like Google style for docstrings:
```suggestion
        Args:
            other: reflection table to match against

        Returns:
            Indices in self, indices in other for matches
```

Reply:
+1

Reply:
I cargo culted in from other local areas; happy to update.